contribs/gnodev/cmd/gnodev/accounts.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"fmt"
+	"log/slog"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/gnolang/gno/contribs/gnodev/pkg/dev"
+	"github.com/gnolang/gno/gno.land/pkg/gnoland"
+	"github.com/gnolang/gno/tm2/pkg/amino"
+	"github.com/gnolang/gno/tm2/pkg/bft/rpc/client"
+	"github.com/gnolang/gno/tm2/pkg/crypto/bip39"
+	"github.com/gnolang/gno/tm2/pkg/crypto/keys"
+	"github.com/gnolang/gno/tm2/pkg/std"
+)
+
+type varAccounts map[string]std.Coins // name or bech32 to coins.
+
+func (va *varAccounts) Set(value string) error {
+	if *va == nil {
+		*va = map[string]std.Coins{}
+	}
+	accounts := *va
+
+	user, amount, found := strings.Cut(value, ":")
```

Comment:
What do you think about keeping the compatibility with existing `balance` sheet formats? Like `<address>=<amount>ugnot`
We use this format for the genesis balance sheets, as well as for the `genesis balances` command